CHANGELOG.md

```diff
@@ -5,12 +5,16 @@ Changelog
 -----
 
 ### Enhancements
-  
+
 - Add support for sending "breadcrumbs" - notable events leading up to an error
   [Christian Schlensker](https://github.com/wordofchristian)
   [#149](https://github.com/bugsnag/bugsnag-js/pull/149)
 
-  
+### Deprecations/Breakages
```

Comment:
@kattrali thanks. Would something like this be more in line with our other repos? Open to suggestions.

```markdown

3.0.0-rc.1 (2016-07-22)
-----

This release adds support for automatic and custom breadcrumb tracking.
Breadcrumbs replace the private implementation of "last event" tracking.
It also removes support for the deprecated `userId` setting, which has been
replaced by the `user` object.

### Enhancements

- Add support for sending "breadcrumbs" - notable events leading up to an error
  [Christian Schlensker](https://github.com/wordofchristian)
  [#149](https://github.com/bugsnag/bugsnag-js/pull/149)

```